### PR TITLE
add vrit tech

### PR DESCRIPTION
**Pull request by kafle1**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

## Why was this change needed?

N/A

## Other information

N/A

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -40,5 +40,19 @@
...
+    "establishedAt": 1546319792,
+    "location": "Sankhamul, Kathmandu",
+    "website": "https://vrittechnologies.com/",
+    "socials": {
+      "facebook": "https://www.facebook.com/vrittech/",
+      "instagram": "https://instagram.com/vrittech",
+      "github": null,
+      "linkedin": "https://www.linkedin.com/company/vrittech/",
+      "twitter": "https://twitter.com/vrittech",
+      "youtube": "https://youtube.com/vrittech"
```
Comment (TheLearneer) [inline data/companies.json:55]: This doesn't seems to working

File: data/companies.json [hunk 2]
```diff
@@ -26,5 +26,19 @@
...
```
Comment (thenishantsapkota) [inline data/companies.json:55]: The YouTube channel doesn't exist.
image

**Review by TheLearneer — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please provide correct YouTube url, if it doesn't exist please set it as `null`.

**Review by TheLearneer — APPROVED** [review]
LGTM.

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution. :confetti_ball: 
Join our [Discord](https://discord.gg/7jwZaa8WDr) to claim your @Contributors role.

**Pull request merged**